config/locales/two_factor_authentication/en.yml

```diff
@@ -19,6 +19,16 @@ en:
     backup_code_header_text: Enter your backup security code
     backup_code_prompt: You can use this security code once. After you enter it,
       you’ll need to use a new key.
+    backup_codes:
+      add_another_authentication_option: '‹ Add another authentication option'
+      instructions: If you don’t have access to another device, keep your backup codes
+        safe. If you lose your backup codes, you won’t be able to sign into
+        %{app_name}.
+      new_backup_codes: I need new backup codes
```

Comment:
I notice in the design that the words "backup codes" break together onto the second line, but it doesn't wrap this way consistently in the implementation.

Mockup|Implementation
---|---
![image](https://github.com/18F/identity-idp/assets/1779930/72fcacc5-1173-4646-bb99-be5df6ce68d6)|![image](https://github.com/18F/identity-idp/assets/1779930/dcfbb53b-428d-4d49-87b7-c3d4da035eaa)

I don't like the idea of forcing a line break since that gets weird with responsiveness, but maybe there's a good case to use a non-breaking space to force "backup codes" to always occur together. We've done this in some other places to ensure that words stay together on the same line (e.g. #7069, [AcuantCapture](https://github.com/18F/identity-idp/blob/23786457658cfd95af062f508d6baa23f93b9da3/app/javascript/packages/document-capture/components/acuant-capture.tsx#L504-L506)).

```suggestion
      new_backup_codes_html: I need new backup&nbsp;codes
```